content/attacks/posts/2019-11-27-Upbit.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2019-11-27
+custodians: Upbit
+categories: "Custodian"
+title: "Upbit Loses $49 Million in Cyberattack"
+---
+
+## Summary
+
+On November 27, 2019, Upbit, one of South Korea's largest cryptocurrency exchanges, fell victim to a major cyberattack, resulting in [the theft of approximately $49 million](https://www.coindesk.com/markets/2019/11/27/upbit-is-the-seventh-major-crypto-exchange-hack-of-2019/#:~:text=Upbit%20is%20the%20latest%20hacking,for%20at%20least%20two%20weeks.) worth of Ethereum (342,000 ETH). The attack specifically targeted Upbit's hot wallet, from which the funds were transferred to an unknown wallet address. To safeguard the remaining assets, Upbit promptly [moved them to cold storage](https://www.coindesk.com/markets/2020/01/14/upbit-exchange-resumes-ether-services-months-after-49m-hack/) as a safety measure. The exchange took immediate action, [replenishing the stolen funds with its own assets](https://upbit.com/service_center/notice?id=1085) and ensuring no impact on its customers.
+
+## Attackers
+
+The identity of the attackers involved in the Upbit hack remains unknown. However, the wallet address associated with the attackers is: [0xa09871AEadF4994Ca12f5c0b6056BBd1d343c029](https://etherscan.io/address/a09871aeadf4994ca12f5c0b6056bbd1d343c029).
+
+## Losses
+
+The Upbit hack resulted in the loss of approximately **$49 million** worth of Ethereum (ETH). 
+
+## Timeline:
+
+- **November 27, 2019, 4:06 (UTC):** Upbit [detects a significant security breach](https://upbit.com/service_center/notice?id=1085) targeting its Ethereum hot wallet, leading to the loss of approximately $49 million worth of Ethereum (342,000 ETH)
```

Comment:
Remove source that's already in the article, also remove second part of the sentence, and simplify the rest. Use factual sentences in Timeline please.

Reply:
Done.

```suggestion
- **November 27, 2019, 4:06 (UTC):** Upbit detects a major security breach in its Ethereum hot wallet.
```